sm_modified_cnn/README.md

```diff
@@ -67,7 +67,8 @@ You should you see the following tree:
 To create the dataset:
 ```bash
 cd Castor/sm_modified_cnn/
-bash create_dataset.sh 
+chmod +x create_dataset.sh
```

Comment:
What I mean is before you commit the script, you make the script executable. Git can track file permissions too besides the content of the file. This way people can just look it's script and run it like `./create_dataset.sh`.